# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/libpciaccess/hotfix-delete-mtrr.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- libpciaccess-0.18/src/linux_sysfs.c.vanilla	2024-03-11 16:00:36.003705156 +0100
+++ libpciaccess-0.18/src/linux_sysfs.c	2024-03-11 16:45:50.784443577 +0100
@@ -731,7 +731,7 @@
 	if (ioctl(pci_sys->mtrr_fd, MTRRIOC_DEL_ENTRY, &sentry) < 0) {
 	    /* FIXME: Should we report an error in this case?
 	     */
-	    fprintf(stderr, "error setting MTRR "
+	    fprintf(stderr, "error deleting MTRR "
 		    "(base = 0x%016" PRIx64 ", size = 0x%08x, type = %u) %s (%d)\n",
 		    (pciaddr_t)sentry.base, sentry.size, sentry.type,
 		    strerror(errno), errno);
